trackmania.js

```diff
@@ -379,6 +379,20 @@ export class TrackmaniaExchangeService extends BaseService {
     }
 }
 
+function toBase64(groupUid, timestamp = undefined) {
```

Comment:
Probably for a future PR, but maybe we can have an `encodeId` and `decodeId` for each of the ID keys you've created (`lb` and `track`). `encodeId` can return the encoded string form, and `decodeId` can return a JSON object that you can unpack into named variables (mostly for sanity of other people cus I can't follow what parameter lives at what index in `args` xd)